hey.go

```diff
@@ -190,7 +196,36 @@ func main() {
 			usageAndExit(err.Error())
 		}
 	}
+	// at this point we will read through the CSV and then use that as the url
+
+	csvfile, err := os.Open(fileName)
+	if err != nil {
+		log.Fatalln("couldnt open the csv file", err)
+	}
+	r := csv.NewReader(csvfile)
+	// TODO remove this after i'm done testing
+	break_early := true
+	for {
+		record, err :=  r.Read()
```

Comment:
Need to check `err` before accessing `record`, which could be empty.
And make sure you handle EOF so you exit gracefully.